authentication/src/main/java/io/camunda/authentication/config/WebSecurityConfig.java

```diff
@@ -47,22 +51,25 @@ public class WebSecurityConfig {
   public static final String LOGIN_URL = "/login";
   public static final String LOGOUT_URL = "/logout";
 
-  public static final String[] UNAUTHENTICATED_PATHS =
-      new String[] {
-        "/login",
-        "/logout",
-        // endpoint for failure forwarding
-        "/error",
-        // all actuator endpoints
-        "/actuator/**",
-        // endpoints defined in BrokerHealthRoutes
-        "/ready",
-        "/health",
-        "/startup",
-        // deprecated Tasklist v1 Public Endpoints
-        "/v1/external/process/**",
-        "/new/**"
-      };
+  public static final Set<String> UNAUTHENTICATED_PATHS =
+      Set.of(
+          LOGIN_URL,
+          "/",
+          // identity is a single page app
+          "/identity/**",
```

Comment:
❓ Would we need this for all applications? I think current the other apps are still functioning and are single page applications

Reply:
I see, thank you for the context